plugins/services/src/js/components/HighOrderServiceConfiguration.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import Pod from "../structs/Pod";
+import PodConfigurationContainer
+  from "../containers/pod-configuration/PodConfigurationContainer";
+import Service from "../structs/Service";
+import ServiceConfigurationContainer
+  from "../containers/service-configuration/ServiceConfigurationContainer";
+
+const HighOrderServiceConfiguration = props => {
```

Comment:
I assume you're wrapping them to provide a constant/simplified interface thus I'd say a name like `GeneralServiceConfigurationContainer` isn't wrong though I think there are propbably better names. BTW: I'd this is a facade not a higher-order component. 

I suggest that we double check if this is the right architecture pattern (including our extensive use of container), but that shouldn't block this PR. I'm fine if you're not addressing the name, but